RobotTask/tests/example.robot

```diff
@@ -0,0 +1,113 @@
+*** Settings ***
+Library    SeleniumLibrary
+Library    String
+Suite Setup    Register New User
+Test Teardown    Close Browser
+
+*** Variables ***
+${BASE_URL}    https://www.demoblaze.com/
+${PRODUCT_PRICE}
+
+*** Test Cases ***
+LoginTest
+    Open Browser    ${BASE_URL}    chrome
+    Wait Until Element Is Visible    id:login2    5s
```

Comment:
Please avoid using locators directly in test cases. Create separate keywords with descriptive names that encapsulate the locators. This improves maintainability and makes tests more readable. If locators change, you only need to update them in one place.